CMakeLists.txt

```diff
@@ -88,8 +88,13 @@ option(WITH_OPTIM "Build with optimisation" ON)
 option(WITH_NEW_STRATEGIES "Use new strategies" ON)
 option(WITH_NATIVE_INSTRUCTIONS
     "Instruct the compiler to use the full instruction set on this host (gcc/clang -march=native)" OFF)
-if(BASEARCH_ARM_FOUND)
-    option(WITH_ACLE "Build with ACLE CRC" ON)
+if(BASEARCH_X86_FOUND)
+    option(WITH_AVX2 "Build with AVX2" ON)
+    option(WITH_SSE2 "Build with SSE2" ON)
+    option(WITH_SSE4 "Build with SSE4" ON)
+    option(WITH_PCLMULQDQ "Build with PCLMULQDQ" ON)
+elseif(BASEARCH_ARM_FOUND)
```

Comment:
I see no reason why BASEARCH_ARM_FOUND shouldn't be the first as A is first letter in Latin alphabet.

Reply:
Good catch. I have corrected it.